README.md

```diff
@@ -0,0 +1,6 @@
+## Project Related Documents:
+https://www.notion.so/Take-Me-To-The-Shop-eb1d65921e8d4e189f9392f9cc7b7a9f
+The above notion document currently does not have any edit rights. Whoever wants to get involved in the project please ping me personally with their email address, I'll add them as a editor for the above notion document.
```

Comment:
```suggestion
The above notion document is publicly open to all and anyone can edit. Hence please add / remote things responsibly so that we don't loose out on anything important.
```